data/hai outfits.txt

```diff
@@ -367,7 +367,7 @@ outfit `"Basrem" Atomic Thruster`
 	"flare sprite" "effect/atomic flare/tiny"
 		"frame rate" 14
 	"flare sound" "atomic tiny"
-	description "Unbeknownst to the scientists of the Deep, the Hai have been using atomic engines for many millennia. Hai atomic engines are still much more energy hungry than other engines, but are more efficient than their human counterparts."
+	description "Unbeknownst to the scientists of the Deep, the Hai have been using atomic engine technology for many millennia. Hai atomic engines are still much more energy-hungry than other engines, but exhibit greater efficiency than their human counterparts."
```

Comment:
No hyphen. [http://www.phrasemix.com/phrases/something-is-energy-efficient](url)